docs/index.md

```diff
@@ -163,7 +163,7 @@ Now, when you run `docker-compose up`, Compose will pull a Redis image, build an
 
 If you're using [Docker Machine](https://docs.docker.com/machine), then `docker-machine ip MACHINE_VM` will tell you its address and you can open `http://MACHINE_VM_IP:5000` in a browser.
 
-If you're not using Boot2docker and are on linux, then the web app should now be listening on port 5000 on your Docker daemon host. If http://0.0.0.0:5000 doesn't resolve, you can also try localhost:5000.
+If you're not using Boot2docker and are on Linux, then the web app should now be listening on port 5000 on your Docker daemon host. If http://0.0.0.0:5000 doesn't resolve, you can also try http://localhost:5000.
```

Comment:
@charleswhchan We might want to be clear here and say if you are not using the `boot2docker.iso` --- that way it isn't confused with the old client.